Dojo.py

```diff
@@ -0,0 +1,425 @@
+from prettytable import PrettyTable
+import sqlite3
+import string
+import os
+import sys
+from subprocess import call
+import shlex
+from Person import *
+
+
+class Dojo:
+    def __init__(self):
+        self.dojo_name = "Kenya Dojo"
+        self.all_rooms = []
+        self.all_people = []
+
+    # Checks whether the room_name you're trying to register is already existent
+    def room_already_registered(self, room_name):
+        available = [room for room in self.all_rooms if room.room_name == room_name]
```

Comment:
naming rooms already exist `available` could get confusing. Its better to use a more descriptive name.